Cargo.toml

```diff
@@ -40,7 +40,7 @@ libc = "0.2"
 log = "0.4"
 libgit2-sys = "0.7.5"
 num_cpus = "1.0"
-rustfix = "0.4"
+rustfix = { git = 'https://github.com/alexcrichton/rustfix', branch = 'rustfix-incremental' }
```

Comment:
Note that this is waiting on https://github.com/rust-lang-nursery/rustfix/pull/140 to be merged and published, so we'll want to hold off on merging this PR until that happens.

Reply:
now published!